common/views/components/Table/Table.tsx

```diff
@@ -106,20 +110,19 @@ const TableTd = styled(Space).attrs({
   as: 'td',
   v: {size: 's', properties: ['padding-top', 'padding-bottom']},
   h: {size: 's', properties: ['padding-left', 'padding-right']},
-})``;
-
-type CellData = {
-  items: string[];
-}
+})`
+  vertical-align: top;
+`;
 
-type TableRow = CellData & {
+type TableRow = {
+  items: (string | JSX.Element | JSX.Element[])[];
   hasHeader: boolean;
 }
 
 type Props = {
-  rows: CellData[];
+  rows: (string | JSX.Element | JSX.Element[])[][];
```

Comment:
Why an element array?

Reply:
good question – it shouldn't be an array. Will update.